.swiftlint.yml

```diff
@@ -1,2 +1,49 @@
+# These are documented at https://github.com/realm/SwiftLint/blob/master/Rules.md
 disabled_rules:
   - colon
+
+  # The following rules may conflict with the PNG project style:
+
+  # (Rules with an asterisk would have been disabled by Workspace anyway if this file were not present,
+  # but SwiftLint interprets this file based on its own defaults.)
+
+  # Would require shorter lines, functions and files.
+  - line_length # *
+  - function_body_length # *
+  - file_length # *
+  - type_body_length # *
+
+  # Would prohibit short identifiers like “t” or “V2”.
+  - identifier_name # *
+  - type_name # *
+
+  # Would require standardized whitespace and indentation.
+  - comma
+  - opening_brace
+  - statement_position # Effectively another brace rule.
+  - closure_parameter_position # Effectively another brace rule.
+  - switch_case_alignment
```

Comment:
is this the style rule that says `switch` should be aligned with all its `case`s, instead of indented out?

Reply:
Yes. [SwiftLint’s own documentation of this rule](https://github.com/realm/SwiftLint/blob/master/Rules.md#switch-and-case-statement-alignment) contains examples.

Reply:
That answered your question, right? (Un‐resolve this if it wasn’t satisfactory.)